projects/packages/my-jetpack/_inc/components/stats-section/count-comparison-card.jsx

```diff
@@ -0,0 +1,95 @@
+import { numberFormat } from '@automattic/jetpack-components';
+import { Card } from '@wordpress/components';
+import { arrowDown, arrowUp, Icon } from '@wordpress/icons';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './style.module.scss';
+
+const subtract = ( a, b ) => {
+	return a === null || b === null || b === undefined ? null : a - b;
+};
+
+export const percentCalculator = ( part, whole ) => {
+	if ( part === null || whole === null || whole === undefined ) {
+		return null;
+	}
+	// Handle NaN case.
+	if ( part === 0 && whole === 0 ) {
+		return 0;
+	}
+	const answer = ( part / whole ) * 100;
+	// Handle Infinities.
+	return Math.abs( answer ) === Infinity ? 100 : Math.round( answer );
+};
+
+/**
+ * CountComparisonCard component.
+ *
+ * @param {object} props                   - Component props.
+ * @param {number} props.count             - Current count.
+ * @param {number} props.previousCount     - Previous count.
+ * @param {React.ReactNode} props.icon     - Icon to display.
+ * @param {React.ReactNode} props.heading  - Card heading.
+ * @returns {object} CountComparisonCard React component.
+ */
+const CountComparisonCard = ( { count, previousCount, icon, heading } ) => {
+	const difference = subtract( count, previousCount );
+	const differenceMagnitude = Math.abs( difference );
+	const percentage = Number.isFinite( difference )
+		? percentCalculator( Math.abs( difference ), previousCount )
+		: null;
+	const shortenedNumberConfig = { maximumFractionDigits: 1, notation: 'compact' };
+
+	return (
+		<Card className={ styles[ 'stats-card' ] }>
+			{ icon && <div className={ styles[ 'stats-card-icon' ] }>{ icon }</div> }
+			{ heading && <div className={ styles[ 'stats-card-heading' ] }>{ heading }</div> }
+			<div className={ styles[ 'stats-card-count' ] }>
+				<span
+					className={ styles[ 'stats-card-count-value' ] }
+					title={ Number.isFinite( count ) ? String( count ) : undefined }
+				>
+					{ numberFormat( count, shortenedNumberConfig ) }
+				</span>{ ' ' }
+				{ difference !== null ? (
+					<span
+						className={ classNames( styles[ 'stats-card-difference' ], {
+							[ styles[ 'stats-card-difference--positive' ] ]: difference < 0,
+							[ styles[ 'stats-card-difference--negative' ] ]: difference > 0,
```

Comment:
What if difference is 0?

Reply:
Then it doesn't have to add the class. It remains with the same style